VDF.GUI/Utils/ImageUtils.cs

```diff
@@ -41,8 +39,20 @@ static class ImageUtils {
 				}
 				g.Save();
 			}
-			img.Save(file, img.RawFormat);
-			return new Bitmap(file);
+			
+			if (persistent) {
+				string file = Core.Utils.FileUtils.SafePathCombine(ThumbnailDirectory, Path.GetRandomFileName() + ".jpeg");
+				img.Save(file, img.RawFormat);
+				return new Bitmap(file);
+			}
+			else {
+				using (MemoryStream memory = new MemoryStream())
```

Comment:
Simply the using

Reply:
I use this position as a representative for others of the type.

Let me put it this way:
From my side it would be completely OK if you adjust such little things yourself. So in the sense that it's not a problem for me if you only take parts, see my code only as a suggestion or especially non-functional adjustments without major coordination simply do it yourself.
At least in terms of time, it costs us both more if you write something that you already have a concrete idea of how it should be, I then sometimes have to find out what you might want to change, I then make the change that you want.... and three weeks later the line is finally so that you accept it.